### PR TITLE
Api app

### PR DESCRIPTION
**Pull request by gone**

### CODE REVIEW

**Comment by gone** [comment]
Needs to get rebased - I took a try this weekend and wasn't able to ceanly rebase it

**Comment by wismer** [comment]
@gone Are we not using stock django for user registration? I had the assumption that we weren't, but can't remember why. Nearly all the merge conflicts from rebasing off of develop involve that module of django.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 from django.core.urlresolvers import reverse
 from django.contrib.auth.hashers import make_password
 
-{% if cookiecutter.django_registration == 'y' %}
+{%- if cookiecutter.django_registration == 'y' -%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Pull out one side of the -'s here - they'll end up con strings

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 from .forms import RegistrationForm
-{% endif %}
+{%- endif -%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Ditto

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ def test_create_user(self):
         user = User.objects.create_user("jonnyrico@fednet.gov", password="iwanttoknowmore")
         User.objects.get(id=user.id)
 
-{%- if cookiecutter.django_registration == 'y' %}
+{%- if cookiecutter.django_registration == 'y' -%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Ditto

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 4]
```diff
@@ -74,7 +74,7 @@ def test_user_registration_works(self):
         self.assertRedirects(actual, expected)
         User.objects.get(email=self.user.email)
 
-{%- endif %}
+{%- endif -%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Ditto

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 5]
```diff
@@ -112,29 +112,6 @@ def test_login_works(self):
 
 {%- if cookiecutter.django_registration == 'y' %}
 
-    def test_register_login_flow_works(self):
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Leave this test in

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py [hunk 6]
```diff
@@ -112,29 +112,6 @@ def test_login_works(self):
...
-                          "password": "bugssuck",
-                          }
-
-        self.client.logout()
-        url = reverse('auth_login')
-        actual = self.client.post(url, self.form_data)
-        expected = reverse("home")
-        self.assertRedirects(actual, expected)
-
-{%- endif %}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/tests.py]: Unclosed If

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py [hunk 7]
```diff
@@ -9,24 +9,4 @@
     url(r'^register/user/', RegistrationView.as_view(), name='user-register'),
     url(r'^password/reset/$', PasswordResetView.as_view(), name='password-reset'),
     url(r'^password/reset/confirm/$', PasswordResetConfirmView.as_view(), name='password-confirm'),
-{%- endif -%}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py]: Unclosed If

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py [hunk 8]
```diff
@@ -9,24 +9,4 @@
...
-urlpatterns = [
-    url(r'^register/$', RegistrationView.as_view(), name='registration_register'),
-]
-
-urlpatterns += [
-    url(r'', include('registration.backends.simple.urls')),
-]
-
-
-{% endif %}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py]: Unclosed if

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py [hunk 9]
```diff
@@ -9,24 +9,4 @@
     url(r'^register/user/', RegistrationView.as_view(), name='user-register'),
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/urls.py]: These views don't have anything to do with djoser - so the preceding if probably doesn't make sense

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/views.py [hunk 10]
```diff
@@ -78,9 +78,5 @@ def register(self, form, **cleaned_data):
         new_user = super().register(form, **cleaned_data)
         return new_user
 
-    def get_success_url(self, user):
-        """
-        Return the url a user should be redirected to after registration
-        """
-        return self.request.GET.get('next', '/')
-{% endif %}
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/views.py]: Unclosed if

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/views.py [hunk 11]
```diff
@@ -78,9 +78,5 @@ def register(self, form, **cleaned_data):
         new_user = super().register(form, **cleaned_data)
         return new_user
 
-    def get_success_url(self, user):
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/views.py]: This is good logic - should stay

**Comment by wismer** [comment]
@gone Maybe we should close this for the time being and reopen when we aren't busy?

**Pull request merged**